specifications/barriers/README.md

```diff
@@ -0,0 +1,47 @@
+# Barrier synchronization
+
+## Barrier.tla
+
+A specification of an abstract Barrier.
```

Comment:
What is a barrier? This is the sort of thing I meant when I asked you to greatly expand the README. It should be clear to someone with no familiarity with you & your work what this spec is about.

Reply:
How about a link: https://en.wikipedia.org/wiki/Barrier_(computer_science)

Reply:
Wikipedia links are good; a link to the published master's thesis would also be helpful.